### ISSUE TITLE
Orange is sort of greenish

### ISSUE THREAD

**Issue by cmsj** [tylerszabo/RGB-Fusion-Tool#47]
# Bug Report

Thank you for taking the time to experiment with this tool and report an issue.

## Summary

I have the Orange colour set in the BIOS, but it's too bright, so I run RGBFusionTool at startup to reduce the brightness, but `--static=Orange` also changes the hue to something more greenish.

### Steps and Commands

`--static=Orange --brightness=30`

### Actual behavior

Colour is a sort of greenish hue

### Expected behavior

Colour is orange

## Environment

Windows 10
Aorus B450 I Pro Wifi

**Comment by tylerszabo** [comment]
Interesting, could you experiment with an RGB value that looks more orange? I just use the default mappings from .NET ([Color.Orange](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.color.orange?view=netframework-4.6)) which is `FF,A5,00` (Note that it does have a fair bit of green in it.)

It would make some sense to remap some of these color names to adjust for the color map of the LEDs (hopefully they use similar LED configurations on various boards so this doesn't need to be per-board).

**Comment by cmsj** [comment]
@tylerszabo oh, I had no idea I could specify RGB values! (I'm not sure that's documented anywhere? Maybe I missed it).

If it helps, these are the RGB values of the pre-defined colours in the BIOS:

White: 255/255/255
Red: 255/0/0
Green: 0/255/0
Blue: 0/0/255
Cyan: 0/255/255
Magenta: 255/0/255
Yellow: 255/216/0
Orange: 255/106/0

(so their Orange is a fair bit different to .Net's)

Thanks very much!

**Comment by tylerszabo** [comment]
Good idea, I've created Issue #48 to track documentation deficiencies.

**Comment by impatman** [comment]
Running into a similiar issue. LED Strips are coming out with the right colors but the on-board leds are off. Trying to set it as purple and it's coming out red. Played around with hex values and green worked fine, blue is very dull, orange comes out dull. Running against a x470.
